lib/gui_rpc_client_print.cpp

```diff
@@ -160,26 +160,37 @@ void RESULT::print() {
     foo = (time_t)report_deadline;
     printf("   report deadline: %s", ctime(&foo));
     printf("   ready to report: %s\n", ready_to_report?"yes":"no");
-    printf("   got server ack: %s\n", got_server_ack?"yes":"no");
```

Comment:
Was the line printf("   got server ack: %s\n", got_server_ack?"yes":"no"); intentionally dropped?